docs/designers-developers/developers/tutorials/format-api/1-register-format.md

```diff
@@ -0,0 +1,49 @@
+# Register a new format
+
+The first thing you're going to do in this tutorial is to register the new format that the plugin intends to apply. WordPress has the [`registerFormatType`](/packages/rich-text/README.md#registerFormatType) primitive to do so.
+
+Let's prepare a minimal plugin to make this work. Create a new folder and a file named `my-custom-format.php` within it containing the necessary PHP code to register and enqueue the JavaScript assets:
```

Comment:
Didn't one of @mkaz's tutorials cover setting up a basic plugin? Might be nice to link out to that.

Reply:
Yeah, the JS tutorial goes into detail about this. It's already referenced a bit before.